sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -3,12 +3,14 @@
 ## 1.10.0-beta.2 (Unreleased)
 
 ### Features Added
+- Added `BrowserCustomizationOptions` to `InteractiveBrowserCredentialBuilder` to allow for customization of the browser window.
 
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
+- Renamed `enableSupportLogging` to `enableUnsafeSupportLogging`
```

Comment:
Is this a breaking change compared to the previous beta version? If so, let's just make it clear here.

Reply:
will do.